package.json

```
@@ -41,6 +41,7 @@
 		}
 	},
 	"devDependencies": {
+		"@aws-sdk/credential-providers": "^3.787.0",
```

I needed to add an extra dependency to get this to run:
```suggestion
		"@aws-sdk/credential-providers": "^3.787.0",
		"@aws-sdk/client-s3": "^3.787.0",
```